.github/actions/spelling/expect/expect.txt

```diff
@@ -1,2878 +1,2880 @@
-USEFILLATTRIBUTE
-USEGLYPHCHARS
-USEHICON
-USEPOSITION
-userbase
-USERDATA
-userdpiapi
-username
-Userp
-userprivapi
-userprofile
-USERSRV
-USESHOWWINDOW
-USESIZE
-USESTDHANDLES
-ushort
-USRDLL
-utf
-utils
-utr
-uuid
-uuidof
-uuidv
-UVWX
-UVWXY
-UWA
-uwp
-uxtheme
-vals
-Vanara
-vararg
-vbproj
-vclib
-Vcount
-vcpkg
-vcprintf
-vcproj
-vcrt
-vcvarsall
-vcxitems
-vcxproj
-vec
-VERCTRL
-versioning
-VERTBAR
-vf
-VFT
-vga
-vgaoem
-viewkind
-viewports
-Virt
-VIRTTERM
-Virtualizing
-vk
-vkey
-VKKEYSCAN
-VMs
-VPA
-VPATH
-VPR
-VPrintf
-VProc
-VRaw
-VREDRAW
-vsc
-vscprintf
-VSCROLL
-vsinfo
-vsnprintf
-vso
-vspath
-vsprintf
-VSTAMP
-vstest
-VSTS
-VSTT
-vstudio
-vswhere
-vtapp
-VTE
-VTID
-vtio
-vtmode
-vtpipeterm
-vtpt
-vtrenderer
-VTRGB
-VTRGBTo
-vtseq
-vtterm
-vttest
-VWX
-waaay
-waitable
-waivable
-WANSUNG
-WANTARROWS
-WANTTAB
-wapproj
-wav
-wbuilder
-wch
-wchar
-WCIA
-WCIW
-WClass
-wcout
-wcschr
-wcscmp
-wcscpy
-WCSHELPER
-wcsicmp
-wcslen
-wcsnicmp
-wcsrchr
-wcsrev
-wcstod
-wcstoul
-wddm
-wddmcon
-wddmconrenderer
-WDDMCONSOLECONTEXT
-wdm
-webpage
-website
-websocket
-wekyb
-WEOF
-wex
-wextest
-wextestclass
-wfdopen
-WFill
-wfopen
-wfstream
-WHelper
-whitelisting
-WIDTHSCROLL
-Widthx
-wiki
-wikia
-wikipedia
-wil
-WImpl
-WINAPI
-winbase
-winbasep
-wincon
-winconp
-winconpty
-winconptydll
-winconptylib
-wincontypes
-WINCORE
-windbg
-WINDEF
-WINDIR
-windll
-WINDOWALPHA
-Windowbuffer
-windowdpiapi
-WINDOWEDGE
-windowext
-WINDOWFRAME
-windowime
-WINDOWINFO
-windowio
-windowmetrics
-WINDOWPLACEMENT
-windowpos
-WINDOWPOSCHANGED
-WINDOWPOSCHANGING
-windowproc
-windowrect
-windowsapp
-windowsinternalstring
-WINDOWSIZE
-windowsx
-WINDOWTEXT
-windowtheme
-WINDOWTITLE
-winevent
-winfx
-wingdi
-winget
-WINIDE
-winioctl
-winmd
-winmeta
-winmgr
-winmm
-winnt
-Winperf
-WInplace
-winres
-winrt
-wintelnet
-winternl
-winuser
-winuserp
-WINVER
-wistd
-wixproj
-wline
-wlinestream
-wmain
-WMSZ
-wnd
-WNDALLOC
-WNDCLASS
-WNDCLASSEX
-WNDCLASSW
-Wndproc
-WNegative
-WNull
-wnwb
-workarea
-workaround
-workflow
-workitem
-wostream
-WOutside
-WOWARM
-WOWx
-wparam
-WPartial
-wpf
-WPR
-WPrep
-WPresent
-wprintf
-wprp
-wprpi
-wregex
-WResult
-writeback
-writechar
-WRITECONSOLE
-WRITECONSOLEINPUT
-WRITECONSOLEOUTPUT
-WRITECONSOLEOUTPUTSTRING
-wrl
-wrp
-WRunoff
-WScript
-wsl
-WSLENV
-wsmatch
-WSpace
-wss
-wstr
-wstring
-wstringstream
-wsz
-wtd
-WTest
-WTEXT
-WTo
-wtof
-wtoi
-WTs
-wtw
-wtypes
-Wubi
-WUX
-WVerify
-wwaproj
-WWith
-wx
-wxh
-xa
-xact
-xamarin
-xaml
-Xamlmeta
-xargs
-xaz
-xbf
-xbutton
-XBUTTONDBLCLK
-XBUTTONDOWN
-XBUTTONUP
-XCast
-XCENTER
-XColors
-xcopy
-XCount
-xdy
-XEncoding
-xes
-Xes
-XES
-xff
-XFile
-XManifest
-XMath
-XMFLOAT
-xml
-xmlns
-xor
-xorg
-XOrg
-Xpath
-XPosition
-XResource
-xsd
-xsi
-xsize
-XSubstantial
-xtended
-xterm
-XTest
-xunit
-xutr
-xvalue
-XVIRTUALSCREEN
-XWalk
-xy
-Xzn
-yact
-YAML
-YCast
-YCENTER
-YCount
-YDPI
-yml
-YOffset
-YPosition
-YSize
-YSubstantial
-YVIRTUALSCREEN
-Yw
-YWalk
-yx
-YZ
-Zc
-ZCmd
-ZCtrl
-zd
-zh
-ZM
-zsh
-zu
-zxcvbnm
-zy
+AAAAAABBBBBBCCC

```

Comment:
Looks like this file got its line endings fragged by the text editor. Mind reverting it and re-applying your changes? Sorry.

Reply:
Since all line ending stuff is a right pain to deal with in git (since it's very opinionated ;P) I'm happy to make this fix if you'd rather.

Reply:
Fwiw, there are also a couple of items in the file that are currently spurious too: https://github.com/microsoft/terminal/pull/1978/checks?check_run_id=1896344914#step:3:115

-- The bot doesn't really care too much about spurious items, it's just that if they reappear later it won't complain about them, since it's expecting them. It assumes they'll eventually be cleaned out the next time it complains about a new unrecognized word. -- Or someone can look at the logs. -- note that the last item has two `words` which will never work, since the syntax is one word per line.

I chose not to have the bot make a comment for cases where no new unrecognized items by default since it felt like that would be too noisy/frustrating -- I'm still considering this choice (amongst many other things). I'm hoping the next version will let users ask the bot to maintain the list automatically -- I might offer to let it mention the maintenance version too. -- I'm currently working on getting this feature to work for private repos (where the APIs I'm trying out do not work well...).

Reply:
Gaaaah... one of my tools is lying to me about line endings. Thanks for pointing it out.